app/views/incidents/new.html.haml

```diff
@@ -16,6 +16,12 @@
         options_from_collection_for_select(@drivers, :id, :proper_name, current_user.id),
         { include_blank: true },
         id: :incident_driver_incident_report_attributes_user_id
+    .field
+      = driver_form.label :occurred_at
+      - a11y_datetime_labels('incident_report_occurred_at').each do |label|
```

Comment:
The argument here `incident_report_occurred_at` gives the wrong labels for those form inputs, I think